### DISCUSSION TITLE
[@types/nova-editor-node] Workspace.activeTextEditor can be undefined

### DISCUSSION THREAD

**Discussion by eablokker** [DefinitelyTyped/DefinitelyTyped#62334] [Issues with a @types package]
@apexskier

Although it's not documented, Workspace.activeTextEditor can be undefined when a non-text-editor tab is active, such as a file browser or terminal tab.

[a comment by an automated account is not shown]

**Comment by apexskier** [comment]
Good find, I'd suggest opening a PR to fix this.

**Reply by eablokker** [reply]
I got Panic to update the documentation to mention this.
Pull Request: #62462
